<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateReceiptVeiw</class>
 <widget class="QWidget" name="CreateReceiptVeiw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,5,8,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Create the receipt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="5,1,5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1,1,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name of the dish:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lE_name">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Enter the cuisine:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lE_cuisine">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Enter the dishtype:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lE_dishtype">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Count of servings:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lE_servings">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Enter calories:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lE_calories">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Enter weight:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lE_weight">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pb_add_photo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Add photo</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,0,8,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Enter ingredients</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="l_hint">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>after cliking on cell its text will be deleted</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tW_ingredients">
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Count</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Ingredients</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pb_add_row">
             <property name="text">
              <string>Add row</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_delete_last_row">
             <property name="text">
              <string>Delete last</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Enter the method of preparing</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="tE_method"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_confirm_receipt">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
